src/main.rs

```diff
@@ -17,12 +17,14 @@ use std::io::Write;
 use std::path::PathBuf;
 
 use commit::{
-    check_staged_files, commit, pre_commit_check, read_cached_commit, write_cached_commit,
+    check_staged_files, commit, git_add_all_modified, pre_commit_check, read_cached_commit,
+    write_cached_commit,
 };
 use commit_message::make_message_commit;
 
 const DEFAULT_CONFIG_FILE: &str = include_str!("../commit-default.json");
 
+#[allow(clippy::pedantic)]
```

Comment:
This is odd, why did you do this?

Reply:
if I dont do this clippy only allows for 2 booleans in the struct so I had to put it in to avoid the clippy error.

Reply:
Add `#[allow(clippy::struct_excessive_bools)]` on top of the file instead

Reply:
line: https://github.com/alt-art/commit/blob/main/src/main.rs#L8